plugins/slack_app/README.md

```diff
@@ -0,0 +1,12 @@
+# Slack App Plugin
+
+This plugin adds a `/deploy` command to Slack which creates Samson deployments.
+To activate:
+
+1. Go over to Slack and create a [new app](https://api.slack.com/apps/new). Many of the fields are completely up to you, but these are important:
+  - Grab the Client ID and secret and verification token, and record them in your environment as `SLACK_CLIENT_ID`, `SLACK_CLIENT_SECRET`, and `SLACK_VERIFICATION_TOKEN`.
+  - Add `https://your.samson.tld/slack_app/oauth` as a redirect URI.
+  - Enable interactive messages, and set the URI to `https://your.samson.tld/slack_app/interact`.
+  - Add a `/deploy` slash-command, and set its URL to `https://your.samson.tld/slack_app/command`.
+2. Visit https://your.samson.tld/slack_app/oauth, and click the "Connect to Slack" button.
+3. Switch back over to Slack, and try out the `/deploy` command!
```

Comment:
So it's failing on [this line](https://github.com/zendesk/samson/blob/216e3c5cf297f5e97dd65713586fab1bb2f4c7c3/plugins/slack_app/app/controllers/slack_app_controller.rb#L56), and it's a bug. If the syntax doesn't match the regex, we should respond with a private message that helps the user figure out how to do it properly. Want to submit a PR with a readme update and a fix?